students/younggeon-yang/user/views.py

```diff
@@ -0,0 +1,77 @@
+import json
+from json.decoder     import JSONDecodeError
+
+from django.http      import JsonResponse
+from django.views     import View
+
+from user.models      import User
+from user.validations import Validation
+from user             import errors
+
+class UserView(View):
```

Comment:
```suggestion
class SignUpView(View):
```
해당 뷰의 이름도 위와 같이 하는 것이 조금 더 명확할 것 같습니다.